charts/flyte-binary/Chart.yaml

```diff
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: v0.1.10 # VERSION
+version: v0.1.11 # VERSION
```

Comment:
This is a magic number I believe. It is set on release. You can just leave this as is.

Reply:
I've rolled this back.